packages/swap/src/routes/__tests__/__snapshots__/quote.test.ts.snap

```diff
@@ -33,7 +33,7 @@ exports[`server GET /quote doesn't include boost information when disabled 1`] =
         "chain": "Bitcoin",
       },
       "fee": {
-        "amount": "200000",
+        "amount": "199800",
```

Comment:
makes sense that pool fee is lower because we deduct ingress fee before calculating it